falcon/status_codes.py

```diff
@@ -14,135 +14,196 @@
-HTTP_507 = HTTP_INSUFFICIENT_STORAGE = '507 Insufficient Storage'
-HTTP_508 = HTTP_LOOP_DETECTED = '508 Loop Detected'
-HTTP_511 = HTTP_NETWORK_AUTHENTICATION_REQUIRED = '511 Network Authentication Required'
+HTTP_500: Final[str] = '500 Internal Server Error'
+HTTP_INTERNAL_SERVER_ERROR = HTTP_500
+HTTP_501: Final[str] = '501 Not Implemented'
+HTTP_NOT_IMPLEMENTED = HTTP_501
+HTTP_502: Final[str] = '502 Bad Gateway'
+HTTP_BAD_GATEWAY = HTTP_502
+HTTP_503: Final[str] = '503 Service Unavailable'
+HTTP_SERVICE_UNAVAILABLE = HTTP_503
+HTTP_504: Final[str] = '504 Gateway Timeout'
+HTTP_GATEWAY_TIMEOUT = HTTP_504
+HTTP_505: Final[str] = '505 HTTP Version Not Supported'
+HTTP_HTTP_VERSION_NOT_SUPPORTED = HTTP_505
+HTTP_507: Final[str] = '507 Insufficient Storage'
+HTTP_INSUFFICIENT_STORAGE = HTTP_507
+HTTP_508: Final[str] = '508 Loop Detected'
+HTTP_LOOP_DETECTED = HTTP_508
+HTTP_511: Final[str] = '511 Network Authentication Required'
+HTTP_NETWORK_AUTHENTICATION_REQUIRED = HTTP_511
 
 # 70X - Inexcusable
-HTTP_701 = '701 Meh'
-HTTP_702 = '702 Emacs'
-HTTP_703 = '703 Explosion'
+HTTP_701: Final[str] = '701 Meh'
+HTTP_702: Final[str] = '702 Emacs'
+HTTP_703: Final[str] = '703 Explosion'
 
 # 71X - Novelty Implementations
-HTTP_710 = '710 PHP'
-HTTP_711 = '711 Convenience Store'
-HTTP_712 = '712 NoSQL'
-HTTP_719 = '719 I am not a teapot'
+HTTP_710: Final[str] = '710 PHP'
+HTTP_711: Final[str] = '711 Convenience Store'
+HTTP_712: Final[str] = '712 NoSQL'
+HTTP_719: Final[str] = '719 I am not a teapot'
 
 # 72X - Edge Cases
-HTTP_720 = '720 Unpossible'
-HTTP_721 = '721 Known Unknowns'
-HTTP_722 = '722 Unknown Unknowns'
-HTTP_723 = '723 Tricky'
-HTTP_724 = '724 This line should be unreachable'
-HTTP_725 = '725 It works on my machine'
-HTTP_726 = "726 It's a feature, not a bug"
-HTTP_727 = '727 32 bits is plenty'
+HTTP_720: Final[str] = '720 Unpossible'
+HTTP_721: Final[str] = '721 Known Unknowns'
+HTTP_722: Final[str] = '722 Unknown Unknowns'
+HTTP_723: Final[str] = '723 Tricky'
+HTTP_724: Final[str] = '724 This line should be unreachable'
+HTTP_725: Final[str] = '725 It works on my machine'
+HTTP_726: Final[str] = "726 It's a feature, not a bug"
+HTTP_727: Final[str] = '727 32 bits is plenty'
 
 # 74X - Meme Driven
-HTTP_740 = '740 Computer says no'
-HTTP_741 = '741 Compiling'
-HTTP_742 = '742 A kitten dies'
-HTTP_743 = '743 I thought I knew regular expressions'
-HTTP_744 = '744 Y U NO write integration tests?'
-HTTP_745 = "745 I don't always test my code, but when I do" 'I do it in production'
-HTTP_748 = '748 Confounded by Ponies'
-HTTP_749 = '749 Reserved for Chuck Norris'
+HTTP_740: Final[str] = '740 Computer says no'
+HTTP_741: Final[str] = '741 Compiling'
+HTTP_742: Final[str] = '742 A kitten dies'
+HTTP_743: Final[str] = '743 I thought I knew regular expressions'
+HTTP_744: Final[str] = '744 Y U NO write integration tests?'
+HTTP_745: Final[str] = (
+    "745 I don't always test my code, but when I do" 'I do it in production'
```

Comment:
It looks like this one got broken earlier (not your fault), could we fix it?

Reply:
Joined into one.